### PR TITLE
Fix docs of oc

### PR DESCRIPTION
**Pull request by jprukner**
There is a fix of https://github.com/openshift/origin/issues/4496 I suggest.

### CODE REVIEW

**Comment by jprukner** [comment]
@bparees @mfojtik @rhcarvalho Please take a look at it and let me know what you think.

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -36,9 +36,8 @@ run new-app:
   $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
 
 This will create an application based on the Docker image 'openshift/ruby-20-centos7' that builds
-the source code at 'github.com/openshift/ruby-hello-world.git'. To start the build, run
-
-  $ %[1]s start-build ruby-hello-world --follow
+the source code at 'github.com/openshift/ruby-hello-world.git'. Build and deployment will be
```
Comment (mfojtik) [inline pkg/cmd/cli/cli.go]: Build will start automatically and deployment will start as soon as the build finishes. (deployment is waiting for an image to deploy ;-)

File: pkg/cmd/cli/cli.go [hunk 2]
```diff
@@ -36,9 +36,8 @@ run new-app:
   $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
 
 This will create an application based on the Docker image 'openshift/ruby-20-centos7' that builds
-the source code at 'github.com/openshift/ruby-hello-world.git'. To start the build, run
-
-  $ %[1]s start-build ruby-hello-world --follow
+the source code at 'github.com/openshift/ruby-hello-world.git'. Build will start automatically and
```
Comment (rhcarvalho) [inline pkg/cmd/cli/cli.go]: "A build will start...", and "a deployment will start..."? :mortar_board:

**Comment by mfojtik** [comment]
LGTM, please squash.

**Comment by rhcarvalho** [comment]
LGTM

**Comment by mfojtik** [comment]
Thanks @jprukner ! 

[merge]

**Comment by bparees** [comment]
[merge]

**Comment by rhcarvalho** [comment]
@bparees can we merge #4528 instead?

**Comment by bparees** [comment]
done.

**Pull request closed without merging**